public/src/components/ComponentsHelpers.ts

```diff
@@ -69,24 +69,25 @@ export const generateRow = (
             row.push(generator(submission, enrollment))
             return
         }
-        row.push("N/A")
+        row.push(Icons.Minus)
     })
     return row
 }
 
-export const generateAssignmentsHeader = (assignments: Assignment[], group: boolean, actions: ReturnType<typeof useActions>, isCourseManuallyGraded: boolean): Row => {
+export const generateAssignmentsHeader = (assignments: Assignment[], viewByGroup: boolean, actions: ReturnType<typeof useActions>, isCourseManuallyGraded: boolean): Row => {
     const base: Row = [
         { value: "Name", onClick: () => actions.setSubmissionSort(SubmissionSort.Name) }
     ]
     if (isCourseManuallyGraded) {
         base.unshift({ value: "ID", onClick: () => actions.setSubmissionSort(SubmissionSort.ID) })
     }
     for (const assignment of assignments) {
-        if (group && assignment.isGroupLab) {
-            base.push({ value: `${assignment.name} (g)`, onClick: () => actions.review.setAssignmentID(assignment.ID) })
-        }
-        if (!group) {
-            base.push({ value: assignment.isGroupLab ? `${assignment.name} (g)` : assignment.name, onClick: () => actions.review.setAssignmentID(assignment.ID) })
+        const cell: CellElement = { value: assignment.name, onClick: () => actions.review.setAssignmentID(assignment.ID) }
```

Comment:
I find the logic below a bit difficult to reason about. I think the following is better. It should be the first statement in the for loop:
```typescript
// If we are viewing by group, ignore all non-group assignments
if (viewByGroup && !assignment.isGroupLab) {
    continue
}
```
Then you can replace the two if statements below with:
```typescript
if (assignment.isGroupLab) {
    cell.iconTitle = "Group"; cell.iconClassName = "fa fa-users"
}